rc/filetype/hbs.kak

```diff
@@ -10,6 +10,7 @@ hook global BufCreate .*[.](hbs) %{
 
 hook global WinSetOption filetype=hbs %{
     require-module hbs
+    require-module html
```

Comment:
I think this should be one of the first thing the module itself does (so inside provide-module) instead of after running it. `require-module hbs` should not rely on the caller also doing `require-module html`.